debian/control

```diff
@@ -3,47 +3,47 @@ Section: graphics
 Priority: optional
 Build-Depends: debhelper (>= 9),
                dh-python,
-               python (>= 2.6.6-3~),
-               python-bs4,
-               python-cairo,
-               python-configobj,
-               python-dbus,
-               python-distutils-extra (>= 2.18),
-               python-gi,
-               python-pil,
-               python-pycurl,
-               python-requests
+               python3,
+               python3-bs4,
+               python3-cairo,
+               python3-configobj,
```

Comment:
@jlu5 Hm, not sure when this was introduced, but we shouldn't need all these libs as build dependencies, right? I guess dh-python, python3 and python3-distutils-extra (>= 2.18) should be enough for building?

Reply:
I'll probably ask about this; removing runtime deps from build-deps makes pybuild complain with a bunch of errors, although the build still seems to be OK.

log snippet:
```
dh binary --with python3 --buildsystem=pybuild
   dh_testroot -O--buildsystem=pybuild
   dh_prep -O--buildsystem=pybuild
   dh_auto_install -O--buildsystem=pybuild
I: pybuild base:217: /usr/bin/python3 setup.py install --root '/build/variety-0.7.0~a1/debian/variety' 
ERROR: Python module cairo not found
ERROR: Python module PIL not found
ERROR: Python module gi not found
ERROR: Python module DominantColors not found
ERROR: Python module Builder not found
ERROR: Python module Util not found
ERROR: Python module bs4 not found
ERROR: Python module requests not found
ERROR: Python module PIL not found
ERROR: Python module gi not found
ERROR: Python module apt_pkg not found
ERROR: Python module gi not found
ERROR: Python module pycurl not found
ERROR: Python module requests.exceptions not found
ERROR: Python module PIL not found
ERROR: Python module helpers not found
ERROR: Python module configobj not found
ERROR: Python module IVarietyPlugin not found
ERROR: Python module helpers not found
ERROR: Python module gi not found
ERROR: Python module PIL not found
ERROR: Python module configobj not found
ERROR: Python module configobj not found
ERROR: Python module dbus not found
ERROR: Python module dbus.service not found
ERROR: Python module dbus.glib not found
ERROR: Python module gi not found
ERROR: Python module PIL not found
running install
```